src/main/java/io/github/sefiraat/crystamaehistoria/listeners/MiscListener.java

```diff
@@ -43,6 +47,16 @@ public void onBlockPlacerStoriedBlock(BlockPlacerPlaceEvent e) {
         }
     }
 
+    @EventHandler
+    public void onShootPaintbrush(EntityShootBowEvent e) {
+        ItemStack itemStack = e.getConsumable();
+        if (SlimefunItem.getByItem(itemStack) instanceof BasicPaintbrush || SlimefunItem.getByItem(itemStack) instanceof InfinitePaintbrush) {
+            e.setCancelled(true);
+            final Entity player = e.getEntity();
```

Comment:
While it’s highly unlikely a skeleton will have a magic paint brush, can we please still have a check here. Call the Entity entity then do if(entity instanceOf Player) {} then send the message :)

Reply:
I actually had that in originally and removed it XD, added back now